bucketlist/src/test/java/io/aexp/bucketlist/HttpBucketListClientTest.java

```diff
@@ -12,12 +12,19 @@
 import io.aexp.bucketlist.data.PullRequest;
 import io.aexp.bucketlist.data.PullRequestActivity;
 import io.aexp.bucketlist.data.PullRequestCommit;
+import io.aexp.bucketlist.data.PullRequestDiff;
+import io.aexp.bucketlist.data.PullRequestDiffResponse;
+import io.aexp.bucketlist.data.PullRequestHunk;
 import io.aexp.bucketlist.data.PullRequestState;
```

Comment:
can you juggle your formatting settings to not separate out the 'java' imports (to match existing style)